taxi/forms.py

```diff
@@ -0,0 +1,52 @@
+from django import forms
+from django.contrib.auth import get_user_model
+from django.contrib.auth.forms import UserCreationForm
+from django.core.exceptions import ValidationError
+
+from taxi.models import Driver, Car
+
+
+class DriverForm(UserCreationForm):
+    class Meta:
+        model = Driver
+        fields = (
+            "username",
+            # "password",
```

Comment:
remove redundant comment. 
You can use `fields = UserCreationForm.Meta.fields + ( <custom_fields>, )`